include/rnp/rnp.h

```diff
@@ -1728,9 +1902,162 @@ RNP_API rnp_result_t rnp_signature_get_creation(rnp_signature_handle_t sig, uint
 RNP_API rnp_result_t rnp_signature_get_expiration(rnp_signature_handle_t sig,
                                                   uint32_t *             expires);
 
+/**
+ * @brief Get the key features if any as per RFC 4880 and later. Do not confuse with key flags.
+ *
+ * @param sig signature handle, cannot be NULL.
+ * @param features on success result will be stored here as or'ed together flag bits.
+ *                 If corresponding value is not available then 0 will be stored.
+ *                 Currently known feature bit flags are (consult RFC for more details):
+ *                 RNP_KEY_FEATURE_MDC  - support for MDC packets (see RFC 4880)
+ *                 RNP_KEY_FEATURE_AEAD - support for OCB encrypted packet and v5 SKESK (please
+ *                   see LibrePGP standard)
+ *                 RNP_KEY_FEATURE_V5   - version 5 public-key format and corresponding
+ *                   fingerprint
+ * @return RNP_SUCCESS or error code if failed.
+ */
 RNP_API rnp_result_t rnp_signature_get_features(rnp_signature_handle_t sig,
                                                 uint32_t *             features);
 
+/**
+ * @brief Get number of the preferred symmetric algorithms, listed in the signature. Applies to
+ *        the self-signature (self-certification or direct-key signature).
+ *
+ * @param sig signature handle, cannot be NULL.
+ * @param count on success nunmber of available algorithms will be stored here. It may be 0 if
+ *              no such information is available within the signature.
+ * @return RNP_SUCCESS or error code if failed.
+ */
+RNP_API rnp_result_t rnp_signature_get_preferred_alg_count(rnp_signature_handle_t sig,
+                                                           size_t *               count);
+
+/**
+ * @brief Get preferred symmetric algorithm from the preferences, specified in the signature.
+ *
+ * @param sig signature handle, cannot be NULL.
+ * @param idx index in the list. Number of availabke items could be obtained via the
```

Comment:
typo (available)

Reply:
Fixed (here, below and issued PR #2272) , thanks.